### PR TITLE
Update examples to use only sdk-core 

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: cookbook/cookbook.md [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
 
 ```js
-import { ProxyNetworkProvider } from "@multiversx/sdk-network-providers";
+import { ProxyNetworkProvider } from "@multiversx/sdk-core";
 
 const proxyNetworkProvider = new ProxyNetworkProvider("https://devnet-gateway.multiversx.com", { clientName: "multiversx-your-client-name" });
 ```
 
-Use the classes from `@multiversx/sdk-network-providers` **only as a starting point**.
+Use the classes from `@multiversx/sdk-core/out/networkProviderss` **only as a starting point**.
```
Comment (andreibancioiu) [inline cookbook/cookbook.md:19]: Typo: `networkProviderss`. `out` should be removed, as well.

File: cookbook/generate.py [hunk 2]
```diff
@@ -60,7 +60,7 @@
 
 :::""",
 
-    "customNetworkProviders": """Use the classes from `@multiversx/sdk-network-providers` **only as a starting point**.
+    "customNetworkProviders": """Use the classes from `@multiversx/sdk-core/out/networkProviderss` **only as a starting point**.
```
Comment (andreibancioiu) [inline cookbook/generate.py:63]: Can we do without `out`?

**Review by andreibancioiu — APPROVED** [review]

File: cookbook/generate.py (hunk 2 above)
Comment (danielailie) [reply cookbook/generate.py]: not at this moment

File: cookbook/cookbook.md (hunk 1 above)
Comment (danielailie) [reply cookbook/cookbook.md]: done

**Review by popenta — APPROVED** [review]

**Pull request merged**